Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -50,7 +50,7 @@ protected function configureRoute(Route $route, \ReflectionClass $class, \Reflec
         // requirements (@Method)
         foreach ($this->reader->getMethodAnnotations($method) as $configuration) {
             if ($configuration instanceof Method) {
-                $route->setMethods(implode('|', $configuration->getMethods()));
+                $route->setMethods(explode('|', implode('|', $configuration->getMethods())));
```

Comment:
In that line it already is a sequence and is https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/380/files#diff-f23783624ea580a4daab1fd95f6a31c0R30 really a supported use case?